### PR TITLE
Added explore and single video page

### PR DESCRIPTION
**Pull request by nishasen**
Explore page contains all the videos listed and and single video page contains video player and some video suggestions related to same category

### CODE REVIEW

**Review by Mohitkumar33 — APPROVED** [review]
You write amazing clean code. With a lot of features

File: vibes/src/Components/VideoPlayer/VideoPlayer.module.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+}
+
+.video_title {
+    width: 100%;
+    font-weight: bold;
+    font-size: 1.2rem;
+}
+
+.video_channel {
+    color: #666666;
```
Comment (shubhamkumar648) [inline vibes/src/Components/VideoPlayer/VideoPlayer.module.css:18]: here you can put value of color  in variable  too for consistency of code

**Review by shubhamkumar648 — APPROVED** [review]
- code looks good understandable

File: vibes/src/Components/VideoPlayer/VideoPlayer.module.css (hunk 1 above)
Comment (nishasen) [reply vibes/src/Components/VideoPlayer/VideoPlayer.module.css:18]: Using that color in one place only so did not create variable for that

**Pull request merged**